templates/stamp/stamp.json

```diff
@@ -0,0 +1,652 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
+            "type": "string",
+            "maxLength": 12
+        },
+        "clusterNameDomainName": {
+            "type": "string"
+        },
+        "osFamilySpec": {
+            "type": "string",
+            "defaultValue": "ubuntu",
+            "allowedValues": [
+                "ubuntu"
+            ]
+        },
+        "jumpBoxVmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D1_v2",
+            "allowedValues": [
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Standard_D1_v2",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14",
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D4_v2",
+                "Standard_D5_v2",
+                "Standard_D11_v2",
+                "Standard_D12_v2",
+                "Standard_D13_v2",
+                "Standard_D14_v2",
+                "Standard_D15_v2",
+                "Standard_DS1_v2"
```

Comment:
Certainly. There isn't a very large price variation between the DS-v2* and others. Depending on how small (resource-wise) we want these jumpboxes, DS* may not cover the lower end of the spectrum. For now, it should be ok to stick with the DS-v2s and if we get push back for under-utilization, we may have to fall back to the standard series or other more suitable options.